bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -288,7 +288,6 @@
 
   /** A flag to decide which implementation to use for data & admin operation */
   public static final String BIGTABLE_USE_GCJ_CLIENT = "google.bigtable.use.gcj.client";
```

Comment:
can we remove?

Reply:
no, its still used by:
https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/BigtableHBaseSettings.java#L43

After beta, when we drop bigtable-client-core, we'll be able to remove it